WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1468,15 +1468,18 @@ TaskInfo                = @ContactInfo@;
 
 # Task List view
 TaskListViewAddNewTaskIcon                                  = display:inline-block; margin-right:12px;
+TaskListViewNewTaskInputWidget                              = z-index:300; overflow:hidden; padding:0px 0px 0px 14px; border-width:0px;
```

Comment:
But I think it should work without `z-index` as well. Can you please check and if it's working then I think we should remove `z-index`.